QuadradiusR/scripts/player_list_row.gd

```diff
@@ -0,0 +1,55 @@
+extends PanelContainer
+
+signal player_hover_start(username)
+signal player_hover_end(username)
+signal player_clicked(username)
+
+export var username = "asd GUEST"
```

Comment:
I think it is better to move all `export var` variables closer to `var` variables as they might server as scene attributes set in constructor. Probably better to have `export var` above standard var`.